src/QMCDrivers/Crowd.h

```diff
@@ -139,6 +144,8 @@ class Crowd
   unsigned long n_accept_          = 0;
   unsigned long n_nonlocal_accept_ = 0;
   /** @} */
+public:
+  WalkerLogCollector wlog_collector_;
```

Comment:
Please make it private and add necessary API for accessing it.

Reply:
Done.

Reply:
it is not obfuscating. Making it public, it can be touched potentially anywhere. I just found out that it is not initialized using "state" in the crowd.